quinn-proto/src/endpoint.rs

```diff
@@ -772,6 +778,7 @@ impl Default for ClientConfig {
     }
 }
 
+/// Internal key for a `Connection` currently open as part of an endpoint
```

Comment:
```suggestion
/// Internal identifier for a `Connection` currently associated with an endpoint
```
"key" and "open" both have more formal meanings elsewhere in QUIC that may confuse things